src/constants/chains.ts

```diff
@@ -1,43 +1,118 @@
 import arbitrumLogoUrl from 'assets/svg/arbitrum_logo.svg'
 import optimismLogoUrl from 'assets/svg/optimism_logo.svg'
 
-export enum SupportedChainId {
+export enum SupportedL2ChainId {
+  ARBITRUM_ONE = 42161,
+  ARBITRUM_RINKEBY = 421611,
+  OPTIMISM = 10,
+  OPTIMISTIC_KOVAN = 69,
+}
+
+export enum SupportedL1ChainId {
   MAINNET = 1,
   ROPSTEN = 3,
   RINKEBY = 4,
   GOERLI = 5,
   KOVAN = 42,
-  ARBITRUM_ONE = 42161,
-  ARBITRUM_RINKEBY = 421611,
-  OPTIMISM = 10,
-  OPTIMISTIC_KOVAN = 69,
+}
+export const SupportedChainId = {
```

Comment:
couldn't this just be a type? `export type SupportChainId = SupportedL2ChainId | SupportedL1ChainId` or the ts equivalent